prow/cluster/BUILD

```diff
@@ -1,18 +1,185 @@
 load("@io_bazel_rules_k8s//k8s:object.bzl", "k8s_object")
+load("@io_bazel_rules_k8s//k8s:objects.bzl", "k8s_objects")
 
 # Usage:
 #   bazel run //prow/cluster:foo.apply
 
 # TODO(fejta): consider k8s_deploy alias
 # https://github.com/bazelbuild/rules_k8s#aliasing-eg-k8s_deploy
 
+k8s_objects(
+    name = "production",
+    objects = [
+        ":branchprotector_cronjob",
+        ":deck_deployment",
+        ":deck_service",
+        ":hook_deployment",
+        ":hook_service",
+        ":horologium_deployment",
+        ":ingress",
+        ":jenkins_deployment",
+        ":jenkins_service",
+        ":lego_deployment",
+        ":mem_limit_range",
+        ":plank_deployment",
+        ":prow_job",
+        ":push_gateway",
+        ":sinker_deployment",
+        ":splice_deployment",
+        ":tide_deployment",
+        ":tide_service",
+        ":tot_deployment",
+        ":tot_service",
+    ],
+)
+
 k8s_object(
     name = "branchprotector_cronjob",
-    cluster = "{STABLE_PROW_CLUSTER}",  # Run make :get-cluster-credentials
+    cluster = "{STABLE_PROW_CLUSTER}",
```

Comment:
nit: consider an alias, this can be defined once in `WORKSPACE` and given basically any name you want (including `k8s_object`)

Reply:
Done